doc/modules/mod_global_distrib.md

```diff
@@ -133,7 +133,7 @@ Global distribution modules expose several per-datacenter metrics that can be us
 
 #### Redis session storage options
 
-* **server** (string, default: `"127.0.0.1"`): Address of the Redis listener.
+* **server** (string, default: `"127.0.0.1"`): Address of the Redis listener. For Unix domain sockets use `{local, "/path/to/listener.sock"}` (available starting with OTP 19).
```

Comment:
`(available starting with OTP 19)` is not necessary, as MongooseIM doesn't support OTP 18 or older. :)

Reply:
Fixed - thanks for noticing.